test_default3-error.chpl:17: error: ambiguous call 'foo()'
test_default3-error.chpl:1: note: candidates are: foo(i: int, j: float, k: string)
test_default3-error.chpl:5: note:                 foo(i: float, j: string, k: int)
